src/lib.rs

```diff
@@ -153,6 +153,10 @@ pub fn middleware(app: Arc<App>) -> MiddlewareBuilder {
     api_router.get("/users/:user_id", C(user::show));
     api_router.get("/users/:user_id/stats", C(user::stats));
     api_router.get("/teams/:team_id", C(user::show_team));
+    api_router.get("/users/:user_id/favorited", C(user::favorited));
+    api_router.put("/users/:user_id/favorite", C(user::favorite));
+    api_router.delete("/users/:user_id/favorite", C(user::unfavorite));
+    api_router.get("/users/:user_id/favorite_users", C(user::favorite_users));
```

Comment:
Are we intending to show the users that another user has favorited? Do these routes actually need the `user_id` in the URL?